app/Http/Controllers/PhotoController.php

```diff
@@ -122,23 +126,25 @@ public function setTitle(PhotoIDsRequest $request, SetTitle $setTitle)
 	/**
 	 * Set if a photo is a favorite.
 	 *
-	 * @param Request $request
+	 * @param PhotoIDsRequest $request
+	 * @param SetStar         $setStar
 	 *
 	 * @return string
 	 */
-	public function setStar(PhotoIDsRequest $request, SetStar $setStar)
+	public function setStar(PhotoIDsRequest $request, SetStar $setStar): string
 	{
-		return $setStar->do(explode(',', $request['photoIDs']), $request['title']) ? 'true' : 'false';
+		return $setStar->do(explode(',', $request['photoIDs'])) ? 'true' : 'false';
 	}
 
 	/**
 	 * Set the description of a photo.
 	 *
-	 * @param Request $request
+	 * @param PhotoIDRequest $request
+	 * @param SetDescription $setDescription
 	 *
 	 * @return string
 	 */
-	public function setDescription(PhotoIDRequest $request, SetDescription $setDescription)
+	public function setDescription(PhotoIDRequest $request, SetDescription $setDescription): string
 	{
 		$request->validate(['description' => 'string|nullable']);
 
```

Comment:
idem

Reply:
At the moment: no. See [this comment](https://github.com/LycheeOrg/Lychee/pull/1101#discussion_r740435600) for an explanation. In the long run, yes. But this is another PR.